cumulusci/tasks/salesforce/ListCommunities.py

```diff
@@ -0,0 +1,42 @@
+from rst2ansi import rst2ansi
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ListCommunities(BaseSalesforceApiTask):
+    api_version = "46.0"
+    task_docs = """
+    Lists Communities for the current org via the Connect API.
+    """
+    task_options = {}
+
+    def _init_options(self, kwargs):
+        super(ListCommunities, self)._init_options(kwargs)
+
+    def _run_task(self):
+        community_list = self.sf.restful("connect/communities")["communities"]
+        communities = [c for c in community_list]
+
+        nameString = "\n==========================================\n"
+
+        communities_output = ["The current Communities in the org are:\n"]
+        for community in communities:
+            communities_output.append("\n{}{}".format(community["name"], nameString))
+            communities_output.append("* **Id:** {}\n".format(community["id"]))
+            communities_output.append("* **Status:** {}\n".format(community["status"]))
+            communities_output.append(
+                "* **Site Url:** `<{}>`_\n".format(community["siteUrl"])
+            )
+            communities_output.append(
+                "* **Url Path Prefix:** {}\n".format(
+                    community.get("urlPathPrefix") or ""
+                )
+            )
+            communities_output.append(
+                "* **Template:** {}\n".format(community["templateName"])
+            )
+            communities_output.append(
+                "* **Description:** {}\n".format(community["description"])
+            )
+
+        communities_output2 = "\n".join(communities_output).encode()
```

Comment:
Please test this out with a community that has non-ascii characters in its name -- and it'd be good to include that in the unit test so that it gets run in both Python 2 and 3. I suspect we'll need to do more work on the encoding here.

Reply:
@davisagli Unfortunately this is where I am out of my depth. Running through the rest of the application to handle encoding appears to be a monumental task. This does seem to only be a Python 2 issue as Python 3 handles it without a problem.

Reply:
@MatthewBlanski I added `from __future__ import unicode_literals` to make sure that string literals are interpreted as unicode in Python 2 (so that interpolated values don't get implicitly encoded as latin1), and specified "utf-8" as the codec to use at the end since it's not the default in Python 2.